vscodesupport/openCode.go

```diff
@@ -0,0 +1,45 @@
+/*Package vscodesupport contains:
+- functionality for opening canaveral projects in vscode
+*/
+package vscodesupport
+
+import (
+	"canaveral/lib"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+)
+
+// OpenCode will take in a project name, and open it in vscode.
+// If such a project doesn't exist, it will return an error.
+// * tested
+func OpenCode(projName string, configPath string) error {
+	if projName == "" {
+		fmt.Println("Please provide a project name.")
+		fmt.Println("(For more info, 'canaveral --help')")
+		return errors.New("No project name specified")
+	} else if !lib.FileExists(configPath) {
+		fmt.Println("No canaveral workspace set. Please specify a workspace.")
+		fmt.Println(
+			"Canaveral needs a workspace to add projects to.")
+		fmt.Println("(For help, type 'canaveral --help')")
+		return errors.New("No canaveral workspace set")
+	}
+	fmt.Printf("Attempting to open Project '%s'\n", projName)
+	ws, err := ioutil.ReadFile(configPath)
+	lib.Check(err)
+	if !lib.DirExists(string(ws) + "/" + projName) {
+		return errors.New("Project '" + projName + "' not found")
+	}
+	err = os.Chdir(string(ws) + "/" + projName)
```

Comment:
I'm not sure it's desirable for the code command to change your current directory to the directory of the folder you're launching in VSCode. Does this currently change that? (Sorry I can't test this locally, I'm reviewing this on a computer that isn't set up to run the code rn)

Reply:
Yea, this shouldn't be a problem. When go runs os.Chdir, it only changes the working directory of the program, not the actual shell. Turns out, this can be somewhat inconvenient in other cases, but for this case, it works well.